bundles/org.openhab.binding.systeminfo/README.md

```diff
@@ -167,6 +167,10 @@ This makes the channels from this groups very flexible - they can change its PID
 
 Parameter PID has a default value 0 - this is the PID of the System Idle process in Windows OS.
 
+## Known issues and workarounds
+
+- Temperature readings are not well supported on standard Windows systems, run OpenHardwareMonitor.exe for the binding to get more reliable readings.
```

Comment:
You cou le add a link to https://openhardwaremonitor.org

Reply:
Done